### ISSUE TITLE
Is there any updates

### ISSUE THREAD

**Issue by rverma-nikiai** [davidmoten/maven-s3-repo#1]
Is this working, can we wrap this with a serveless.yaml

**Comment by davidmoten** [comment]
I'm not up to speed on changes in AWS on this front. As stated on README there was a blocker with their API Gateway constraints. If you find out that it's no longer the case then I might work on this again.

**Comment by rverma-nikiai** [comment]
Api gateway does support binary request/responses now. https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-payload-encodings-configure-with-console.html

There is a cool serverless pip repository which is using the same. https://github.com/khornberg/elasticpypi.

Although there is another project which combines s3 and maven seamlessly, https://github.com/jenkins-x/bucketrepo. I found this after raising this question and it does work well. 

Are you planning to add something over it?

**Comment by davidmoten** [comment]
The problem is wildcard paths, not binary encodings although that is very finicky and I am yet to see good working examples with cloudformation.

I already have a Beanstalk app that proxies S3 for maven. What I want with this project is a serverless version.

**Comment by davidmoten** [comment]
The pypi project don't realise that you can beat the apig download limit with S3 encrypted url redirects.

**Comment by davidmoten** [comment]
I think the concept of using serverless here is still not very practical because of the upload limit. It's very easy to hit 6mb limit on apig +lambda integration and then you need custom wagons in maven as transport.

**Comment by rverma-nikiai** [comment]
I agree, the idea was not to push serverless but to showcase usage of
apigateway with binary media type :)

**Comment by davidmoten** [comment]
Thanks, I need to clean up the README a bit because there are conflicting statements in there about binary support.
